lib/fluent/plugin/in_sql.rb

```diff
@@ -74,7 +74,11 @@ def init(tag_prefix, base_model, router, log)
         # creates a model for this table
         table_name = @table
         primary_key = @primary_key
+        time_format = @time_format
+
         @model = Class.new(base_model) do
+          @@time_format = time_format
```

Comment:
We wouldn't use class variable.
For reloading feature support, please use variable store instead:
https://github.com/fluent/fluentd/blob/master/lib/fluent/variable_store.rb